core/src/env/srml/srml_only/impls.rs

```diff
@@ -145,4 +145,25 @@ where
             )
         }
     }
+
+    fn call(
+        callee : <Self as EnvTypes>::AccountId,
+        gas: u64,
```

Comment:
This is basically correct, however as far as I remember there is a PR in the pipeline to make the type of `gas` generic. Not sure if it has been implemented for ink! yet.

Reply:
Actually saw that this is not the case: https://github.com/paritytech/substrate/pull/2944/files#diff-375b14443b202b601fb667ba014f0711R269

Reply:
I added the gas generic type

Reply:
Have you clicked on the link I provided to you?
The current state of SRML is that even though Gas is meant to be generic we fixed it to u64.
I will ask the SRML side if a generic type for ink! is useful at all given that decision. @pepyakin

Reply:
Talked to @pepyakin and he confirmed that gas is no longer treated as a generic type and ink! should make use of fixed `u64` instead.

Reply:
I misunderstood your intention.
All right, I roll back this change.